code/go/0chain.net/smartcontract/storagesc/stakepool.go

```diff
@@ -460,7 +460,11 @@ func (ssc *StorageSmartContract) stakePoolLock(t *transaction.Transaction,
 	}
 
 	// TO-DO: Update stake in eventDB
-
+	data, _ := json.Marshal(event.UpdateTotalStake{
```

Comment:
Can you reuse ` event.TagUpdateBlobber`. 
```go
	data, err := json.Marshal(&dbs.DbUpdates{
		Id: sn.ID,
		Updates: map[string]interface{}{
			"total_stake":                  sp.stake(),
		},
	})
	if err != nil {
		return fmt.Errorf("marshalling update: %v", err)
	}
	balances.EmitEvent(event.TypeStats, event.TagUpdateBlobber, sn.ID, string(data))
```
This should mean you don't need any `dbs/event` changes.

Also once you have finished `// TO-DO: Update stake in eventDB` can be removed.